lib/webmock/fixtures/manager.rb

```diff
@@ -3,6 +3,103 @@ module WebMock
   module Fixtures
     # Class used to manage WebMock fixtures
     class Manager
+      # Available fixtures which have been loaded
+      @registered_fixtures = {}
```

Comment:
There isn't any documentation for `registered_fixtures` nor `started_fixtures`. Not sure if you wanted some